client/containers/User/User.tsx

```diff
@@ -31,7 +31,7 @@ const User = (props: Props) => {
 	const authoredPubs = communityPubs.filter((pub) => {
 		const collaborators = pub.attributions || [];
 		const isAuthor = collaborators.reduce((prev, curr) => {
-			if (curr.user.id === loginData.id && curr.isAuthor) {
+			if (curr.isAuthor) {
```

Comment:
This will set `isAuthor` to true if _any_ of the collaborators are an author. We need to still only check the `.isAuthor` key for the collaborator that lines up with the current profile. I think that simply means:

```
if (curr.user.id === userData.id && curr.isAuthor) {
```

We could also do this a bit more cleanly now using `.some`:

```
const isAuthor = collaborators.some((collaborator) => {
	return collaborator.user.id === userData.id && collaborator.isAuthor;
});
```

Or - since this seems to have tripped me up 3 years ago, we can be more descriptive with an extra variable:
```
const isAuthor = collaborators.some((collaborator) => {
	const isCurrentProfileCollaborator = collaborator.user.id === userData.id;
	return isCurrentProfileCollaborator && collaborator.isAuthor;
});
```